aion/web/elm/src/Room/View.elm

```diff
@@ -64,21 +73,51 @@ displaySingleScore userRecord =
     ListGroup.li [] [ text (userRecord.name ++ ": " ++ (toString userRecord.score)) ]
 
 
-displayQuestion : String -> Html Msg
-displayQuestion question =
-    Card.config [ Card.attrs [ class "room-question" ] ]
-        |> Card.block [] [ Card.text [] [ text question ] ]
-        |> Card.view
+displayQuestion : String -> RoomState -> Html Msg
+displayQuestion question roomState =
+    let
+        temporaryText =
+            "Get ready, the next question is going to appear soon!"
+
+        ( cardClass, textFieldValue ) =
+            case roomState of
+                QuestionDisplayed ->
+                    ( "room-question", question )
+
+                QuestionBreak ->
+                    ( "room-question-hidden", temporaryText )
+    in
+        Card.config [ Card.attrs [ class cardClass ] ]
+            |> Card.block [] [ Card.text [] [ text textFieldValue ] ]
+            |> Card.view
+
 
+displayQuestionImage : Location -> ImageName -> RoomState -> Html Msg
+displayQuestionImage location imageName roomState =
+    let
+        questionImageSource =
+            getImageUrl location imageName
+
+        imageSource =
+            case roomState of
+                QuestionDisplayed ->
+                    questionImageSource
+
+                QuestionBreak ->
+                    "https://i.ytimg.com/vi/NjlzcriYc8o/maxresdefault.jpg"
```

Comment:
...

Reply:
@pmrukot I needed to add something as a placeholder for the actual thing that we will end up with after the UX/UI redesign, boi seemed like a good temporary solution, as `here come the new question` :smile: